tests/causal-inference-discovery-book/test_causal_inference_discovery_book.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import pandas as pd
+from pytest import mark
+from scipy import stats
+from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.linear_model import LassoCV, LinearRegression, LogisticRegression
+from tqdm import tqdm
+
+from dowhy import CausalModel
+
+# TODO: Add tests for Chapter 6, 7 (already below) + 9, 10, Extras_02
+
+class GPSMemorySCM:
```

Comment:
I'm using the online version of the book, which doesn't have page numbers. But, I think the chapter numbers (which I added) should make it clear since the tests span the entire chapter.